camel/models/stub_model.py

```diff
@@ -26,6 +26,19 @@ def __init__(self, model_type: ModelType,
                  model_config_dict: Dict[str, Any]) -> None:
         r"""All arguments are unused for the dummy model."""
         super().__init__(model_type, model_config_dict)
+        self._token_counter: Optional[BaseTokenCounter] = None
+
+    @property
+    def token_counter(self) -> BaseTokenCounter:
+        r"""Initialize the token counter for the model backend.
+
+        Returns:
+            BaseTokenCounter: The token counter following the model's
+                tokenization style.
+        """
+        if not self._token_counter:
+            self._token_counter = OpenAITokenCounter(self.model_type)
```

Comment:
Please create `StubTokenCounter(BaseTokenCounter)` class in this file that returns some constant (say int 10) from count_tokens_from_messages. We want to keep stubs super lightweight.